### PR TITLE
License files

### PR DESCRIPTION
**Pull request by mcyprian**
Doc files are separated to license files and other doc files, License files are listed in %license section of the spec file instead of %doc.
This PR also contains minor fix of created spec file's name when -s/--srpm option is set.

### CODE REVIEW

**Comment by hroncok** [comment]
More tests with different files? COPYRIGHT etc.?

**Comment by mcyprian** [comment]
Are there any other file names except */LICENSE */COPYRIGHT */COPYING that should be moved to %license?

**Comment by hroncok** [comment]
well, see for example https://github.com/encukou/py3c/blob/master/LICENSE.MIT https://github.com/encukou/py3c/blob/master/doc/LICENSE.CC-BY-SA-3.0 and similar.

File: pyp2rpm/metadata_extractors.py [hunk 1]
```diff
@@ -550,8 +552,11 @@ def data_from_archive(self):
         archive_data['summary'] = self.summary
         archive_data['home_page'] = self.home_page
         doc_files = self.doc_files
-        archive_data['doc_files'] = [doc for doc in doc_files if 'license' not in doc.lower()]
-        archive_data['doc_license'] = [doc for doc in doc_files if 'license' in doc.lower()]
+        archive_data['doc_files'] = [doc for doc in doc_files if all(s not in doc.lower() for s in
+                                                                     settings.LICENSE_FILES)]
+        archive_data['doc_license'] = [doc for doc in doc_files if any(s in doc.lower() for s in
+                                                                       settings.LICENSE_FILES)]
```
Comment (hroncok) [inline pyp2rpm/metadata_extractors.py]: I read this two times. Better to have a function?

**Comment by mcyprian** [comment]
Fixed.

File: pyp2rpm/metadata_extractors.py [hunk 2]
```diff
@@ -355,7 +363,8 @@ def data_from_archive(self):
         archive_data['python_versions'] = py_vers[1:] if py_vers \
             else [settings.DEFAULT_ADDITIONAL_VERSION]
 
-        archive_data['doc_files'] = self.doc_files
+        archive_data['doc_files'], archive_data[
+            'doc_license'] = self.separate_license_files(self.doc_files)
```
Comment (hroncok) [inline pyp2rpm/metadata_extractors.py]: Any chance of some more readable line break? Not necessarily, but this feel, ehm, ugly?

**Comment by hroncok** [comment]
Everything except the cosmetic comment about the line break seems OK.

**Comment by mcyprian** [comment]
I think it looks a bit better now :) thanks for the review @hroncok.

**Pull request merged**